.circleci/prepare.sh

```diff
@@ -1,3 +1,3 @@
 $PYTHON boots.py create --group package
```

Comment:
this may need to change from boots.

Reply:
I think this file and nearby files are the leftovers of a previous CI process and can be removed.